lib/attributes_json.rb

```diff
@@ -15,7 +15,10 @@ module FormatParser::AttributesJSON
 
   # Implements a sane default `as_json` for an object
   # that accessors defined
-  def as_json(root: false)
+  #
+  # stringify_keys: transforms all the hash keys to a string when is true. The
```

Comment:
We have YARD documentation for most methods I think, so the documentation for new parameters should also be in YARD (`@param` and the like)

Reply:
great, I've updated there!